.gitignore

```diff
@@ -0,0 +1,18 @@
+composer.phar
+bin
+config.php
+
+# directories
+vendor/*
+.composer/*
+
+# configuration
+mouf/Mouf.php
+mouf/MoufUI.php
+
+# Phpstorm
```

Comment:
Some people think it is a best practice to put environment related files in a **global** .gitignore
(so .idea and .sublime-workspace should not be in there).
Not a big deal though...